code/entry.py

```diff
@@ -1,79 +1,98 @@
         #     query_qep = 'EXPLAIN (FORMAT JSON) ' + query
-        # elif (command == 'CREATE' and query.split(' ')[1] == 'TABLE'):
+        # elif command == 'CREATE' and query.split(' ')[1] == 'TABLE':
         #     query_qep = 'EXPLAIN (FORMAT JSON) ' + query
         # print(query_qep)
 
         query_array = query.split(';')
         qep_index = 0
         for query in query_array:
-            if query and (not query.isspace()):
-                # print(query)
-                query = query.strip()
-                query_qep = ''
-                command = query.split(' ')[0]
-                if command in {'SELECT', 'INSERT', 'UPDATE', 'DELETE', 'VALUES', 'EXECUTE', 'DECLARE'}:
-                    query_qep = 'EXPLAIN (FORMAT JSON) ' + query
-                elif command == 'CREATE' and query.split(' ')[1] == 'TABLE':
-                    query_qep = 'EXPLAIN (FORMAT JSON) ' + query
-                if len(query_qep) > 0:
-                    cur.execute(query_qep)
-                    rows = cur.fetchall()
-                    for row in rows:
-                        qep_index += 1
-                        with open('../data/json/' + file_name + '_' + str(qep_index) + '.json', 'wt') as out:
-                            res = json.dump(row[0], out, sort_keys=True, indent=4, separators=(',', ': '))
+            if not query or query.isspace():
+                continue
+            print("Query:", query)
+            query = query.strip().lower()
+            query_qep = ''
+            command = query.split(' ')[0]
+            if command in ['select', 'insert', 'update', 'delete', 'values', 'execute', 'declare']:
+                query_qep = 'explain (format json) ' + query
+            elif command == 'create' and query.split(' ')[1] == 'table' and query.split(' ')[2] == "as":
+                query_qep = 'explain (format json) ' + query
+            if len(query_qep) == 0:
+                continue
+            cur.execute(query_qep)
+            rows = cur.fetchall()
+
+            for row in rows:
+                qep_index += 1
+                qep_json_path = os.path.join('..', 'data', 'json', file_name + '_' + str(qep_index) + ".json")
+                qep_text_path = os.path.join('..', 'data', 'txt', file_name + '_' + str(qep_index) + ".txt")
+                with open(qep_json_path, 'wt') as out:
+                    res = json.dump(row[0], out, sort_keys=True, indent=2, separators=(',', ': '))
+                translate_qeptree_to_text(qep_json_path, qep_text_path)
+
+
+
+            print('Executing query...')
+            cur.execute(query)
+            print('Row count: ' + str(cur.rowcount), " - Results:")
+            if cur.rowcount <= 0:
+                print("")
+                continue
+            rows = cur.fetchall()
+            for row in rows:
+                for i in range(len(row)):
+                    print("   ", row[i], end='')
+                print("\n")
```

Comment:
Only need to display results for last queries?

Reply:
psql displays all, so we should display all :))